strat-bot.py

```diff
@@ -1,15 +1,15 @@
 import discord
+from discord.ext import commands
 import random
 from gsheet import Sheet
 import datetime
+# import asyncio
 
 print(discord.version_info)
-
-client = discord.Client()
-user = []
-voices = []
-
 token = open("token.txt","r").readline()
+#This should work? If not, idk
+client = commands.Bot(command_prefix='!', case_insensitive=True, description='memer strat bot.')
```

Comment:
should we make the prefix be `!strat` so it only runs on strat commands? Then just make the `!help` be `!strat help`?

Reply:
You could. But I don't see any other bot using strat as a command. If a command isn't found in a bot you just get 
Ignoring exception in command None:
discord.ext.commands.errors.CommandNotFound: Command "start" is not found

Plus, not sure how it would affect the command invoke